assets/csidriveroperators/openstack-cinder/07_deployment.yaml

```diff
@@ -39,6 +39,10 @@ spec:
         image: ${OPERATOR_IMAGE}
         imagePullPolicy: IfNotPresent
         name: openstack-cinder-csi-driver-operator
+        resources:
+          requests:
+            cpu: 10m
+            memory: 20Mi
```

Comment:
Can you take the chance and use the same values used here?

https://github.com/bertinatto/cluster-storage-operator/blob/bc8d42c103305e23c43b0e87acf700a73bdab28b/assets/csidriveroperators/aws-ebs/07_deployment.yaml#L43-L46